Library/Formula/augeas.rb

```diff
@@ -12,7 +12,6 @@ class Augeas < Formula
 
   head do
     url "https://github.com/hercules-team/augeas.git"
-
```

Comment:
Whitespacing makes me feel happy about reading formulae. It feels more readable. I'll go quietly if others prefer no whitespacing though :wink:.